Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -1765,8 +1766,11 @@
   # icon: { sprite: /Textures/Clothing/OuterClothing/Armor/syndie-raid.rsi, state: icon }
   icon: { sprite: /Textures/SS220/Clothing/OuterClothing/Armor/syndie-raid.rsi, state: icon } #ss220 raid suit resprite
   productEntity: ClothingBackpackDuffelSyndicateRaidBundle
+  discountCategory: veryRareDiscounts
+  discountDownTo:
+    Telecrystal: 6 #ss220-uplink-discounts
   cost:
-    Telecrystal: 8
+    Telecrystal: 12 #ss220-uplink-discounts
```

Рейдерский костюм вырезается из агентского аплинка в PR: https://github.com/SerbiaStrong-220/space-station-14/pull/2646 из-за чего делать его для нюкеров дороже - не имеет смысла. Откатить.
```suggestion
    Telecrystal: 8
```